pyproject.toml

```diff
@@ -7,4 +7,4 @@ order_by_type = "False"
 
 [tool.black]
 line-length = 120
-exclude = "./proto/"
+exclude = "./python/"
```

Comment:
May not be required if using betterproto.

Reply:
Ack.